tests/unit_test/app_common/aggregators/in_time_accumulate_weighted_aggregator_test.py

```diff
@@ -72,11 +72,14 @@ class TestInTimeAccumulateWeightedAggregator:
     )
     def test_invalid_create(self, exclude_vars, aggregation_weights, expected_data_kind, error, error_msg):
         with pytest.raises(error, match=re.escape(error_msg)):
-            _ = InTimeAccumulateWeightedAggregator(
```

Comment:
Move this into another PR

Reply:
Need to have this for unit tests to pass.